### PR TITLE
MM-32347 Site Subsections

### PR DESCRIPTION
**Pull request by anurag6713**
#### Summary
Site Configuration
* Customization
* Localization
* Users and Teams
* Notifications
* Announcement Banner
* Emoji
* Posts
* File Sharing and Downloads
* Public Links
* Notices

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-32347

### CODE REVIEW

**Review by sbishel — APPROVED** [review]

**Review by hahmadia — APPROVED** [review]
Looks great! Thanks

File: src/constants/permissions.ts [hunk 1]
```diff
@@ -141,9 +163,7 @@ const values = {
     SYSCONSOLE_WRITE_ENVIRONMENT_PERFORMANCE_MONITORING: 'sysconsole_write_environment_performance_monitoring',
     SYSCONSOLE_READ_ENVIRONMENT_DEVELOPER: 'sysconsole_read_environment_developer',
     SYSCONSOLE_WRITE_ENVIRONMENT_DEVELOPER: 'sysconsole_write_environment_developer',
-    SYSCONSOLE_READ_SITE: 'sysconsole_read_site',
-    SYSCONSOLE_WRITE_SITE: 'sysconsole_write_site',
-    SYSCONSOLE_READ_AUTHENTICATION: 'sysconsole_read_authentication',
+   SYSCONSOLE_READ_AUTHENTICATION: 'sysconsole_read_authentication',
```
Comment (sbishel) [inline src/constants/permissions.ts:166]:
```suggestion
    SYSCONSOLE_READ_AUTHENTICATION: 'sysconsole_read_authentication',
```

**Review by furqanmlk — APPROVED** [review]

**Pull request merged**